database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -51,10 +51,10 @@
  */
 public abstract class FirebaseListAdapter<T> extends BaseAdapter {
 
+    private Activity mActivity;
     private final Class<T> mModelClass;
-    protected int mLayout;
-    protected Activity mActivity;
-    FirebaseArray mSnapshots;
+    private int mLayout;
+    private FirebaseArray mSnapshots;
```

Comment:
The snapshot array is accessed by the unit tests. For that reason it must be package private/default access.